src/Agent.Worker/TestResults/ResultsCommandExtension.cs

```diff
@@ -188,7 +188,9 @@ private void ProcessPublishTestResultsCommand(IExecutionContext context, Diction
                         isAutomated: true,
                         buildId: runContext != null ? runContext.BuildId : 0,
                         buildFlavor: runContext != null ? runContext.Configuration : string.Empty,
-                        buildPlatform: runContext != null ? runContext.Platform : string.Empty
+                        buildPlatform: runContext != null ? runContext.Platform : string.Empty,
+                        releaseUri: runContext != null ? runContext.ReleaseUri : null,
+                        releaseEnvironmentUri: runContext != null ? runContext.ReleaseEnvironmentUri : null
```

Comment:
is there any specific reason to use null, instead of string.empty, as buildPlatform and buildFlavor are defaulting to string.empty or viceversa